subsys/bluetooth/services/cts_c.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2012 - 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <sys/byteorder.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/uuid.h>
+#include <bluetooth/gatt.h>
+
+#include <bluetooth/services/cts_c.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(cts_c, CONFIG_BT_GATT_CTS_C_LOG_LEVEL);
+
+/** The lowest valid Current Time year is the year when the Western calendar was introduced. */
```

Comment:
`/**` ->  `/*` This file isn't parsed by doxygen

Reply:
Done.